docs/development/go/go-structures/index.md

```diff
@@ -259,7 +557,7 @@ func prettyPrint(s interface{}) {
     fmt.Println(string(p))
 }
 
-type Record struct {
```

Comment:
Renamed Record to Contact since it seemed like it would represent someone's contact in their phonebook